docs/source/installation.rst

```diff
@@ -9,9 +9,9 @@ The following section includes installation instructions for `MHKiT-Python <http
 MHKiT-Python
 -------------
 
-`MHKiT-Python <https://github.com/MHKiT-Software/MHKiT-Python>`_ requires `Python (3.6 or 3.7) <https://www.python.org/>`_ , and has several Python package dependencies.  
-It is recommended to use the `Anaconda Python Distribution <https://www.anaconda.com/distribution/>`_ to install Python, since it includes most of MHKiT-Python's package dependencies.
-Refer to the `Python website <https://www.python.org/>`_ for information on using Python.  
+`MHKiT-Python <https://github.com/MHKiT-Software/MHKiT-Python>`_ requires `Python (3.6 or 3.7) <https://www.python.org/>`_  and has several Python package dependencies.
+It is recommended to use the `Anaconda Python Distribution <https://www.anaconda.com/distribution/>`_ to install Python since it includes most of MHKiT-Python's package dependencies.
+Refer to the `Python website <https://www.python.org/>`_ for information on using Python.
```

Comment:
@SeanPluemer can you add a note about selecting the option to adds python to your Windows Environmental Variables when installing Anaconda. This is no longer the default with Anaconda3. Here’s [some info ](https://www.datacamp.com/community/tutorials/installing-anaconda-windows)and some [more info](https://docs.anaconda.com/anaconda/user-guide/faq/) about the issue

Reply:
@SeanPluemer Could you take care of this?